.changeset/flat-buses-check.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": minor
+---
+
+Added `size` prop to `Card` and `InteractableCard`. The `size` prop changes the padding and can be `small`, `medium` or `large`.
```

Comment:
should this not be paddingSize - size would suggest that the total height and width of the card are changing ?